srcpkgs/khal/template

```diff
@@ -1,27 +1,37 @@
 # Template file for 'khal'
 pkgname=khal
-version=0.11.3
-revision=2
-build_style=python3-module
+version=0.13.0
+revision=1
+build_style=python3-pep517
 # Tests get the path to the configuration file wrong in an xbps-src chroot
 make_check_args="
  --deselect tests/cli_test.py::test_configure_command
  --deselect tests/cli_test.py::test_configure_command_create_vdir
 "
+# Some locale-related failures.
```

Comment:
This is bash, use `+=` to append arguments. Also move this whole block after all of the template variable definitions, separated from the variable block by a blank line.

I'm not worried about one or two failing i686 checks. You can deselect those for that arch if you want.